conf/script/build-ha-csm

```diff
@@ -172,7 +172,7 @@ csm_rsc_add() {
     sudo pcs -f $cib_file resource create csm-agent systemd:csm_agent op \
         monitor interval=30s
     sudo pcs -f $cib_file resource create csm-web systemd:csm_web op \
-        monitor interval=30s
+        monitor interval=30s meta failure-timeout=60s
```

Comment:
@rajanikantchirmade , I think you need to add migration-threshold to it as well because this is a active/passive resource. Pacemaker will try to restart the service  migration-threshold times and then failover the resource then try again on the primary node.
Is this tested on hardware?

Reply:
@mssawant  Yes, it is tested on H/W setup, where it got reproduced.

Reply:
Okay, you need to make sure that pacemaker fails to start csm service and it failsover. Then once pacemaker is able to start the service on earlier node then it csm should automatically failback.
You  can try this,
```
delete csm_agent systemd unit file (backup the original before deleting), then disable and enable csm_agent resource, pacemaker will try to start but will fail, then csm stack will failover. After failover, replace the systemd unit file and see if csm fails back automatically.
```

Reply:
Approving the patch, @rajanikantchirmade please confirm the case as mentioned above.

Reply:
@mssawant you mentioned csm-agent, but it doesn't have any failure timeout. So, i presume it is about csm-web.
If we want to have the same behavior for csm-agent - need to update its meta attributes as well.

Reply:
@azheregelya  Yes, failure-time out needed for csm-agent too. For csm_web I already tested yesterday I am looking for H/W setup to check again failover/failback and for csm-agent too.

Reply:
@azheregelya  Done, added failure timeout for csm_agent too.

@mssawant  migration-threshold has exception if  `start-failure-is-fatal=true` for cluster. It's default value is `true`.
So resource is always migrate in this case.

```
[root@inteln13-m07 ~]# pcs property list --all | grep start-failure-is-fatal
 start-failure-is-fatal: true
[root@inteln13-m07 ~]#
```